README.rst

```diff
@@ -173,5 +176,115 @@ Github Actions is configured to run unit tests on every new PR. The tests are co
 .. _jwt: https://jwt.io
 
 
+Install as a Django app
+-----------------------
+
+Add to your `requirements.txt`:
```

Comment:
Is it possible to set these up directly as requirements, so that if you add the CatchPy package it will install these?

Reply:
Good call; in fact, this is already the case but the package dependencies aren't pinned.